src/python/RucioUtils.py

```diff
@@ -116,25 +116,34 @@ def getRuleQuota(rucioClient=None, ruleId=None):
 
 def getRucioUsage(rucioClient=None, account=None, activity =None):
     """ size of Rucio usage for this account (if provided) or by activity """
-    filters = {'activity': activity}
-    
-    if account is not None and account != 'crab_tape_recall':
-        filters['account'] = account
-    
-    rules = rucioClient.list_replication_rules(filters=filters)
-
-    if activity == 'Analysis Input':
-        valid_states = ['OK', 'REPLICATING', 'STUCK', 'SUSPENDED']
-    elif activity == 'Analysis TapeRecall':
-        valid_states = ['REPLICATING', 'STUCK', 'SUSPENDED']  # Exclude 'OK' state
+    if activity is None:
+        if account is None:
+            print("Error: Account and Activity both unspecified")
+            totalusage = 0
+        else:
+            usageGenerator = rucioClient.get_local_account_usage(account=account)
+            totalBytes = 0
+            for usage in usageGenerator:
+                used = usage['bytes']
+                totalBytes += used
+            totalusage = totalBytes
     else:
```

Comment:
let's add a comment line here, on the tune of  "Rucio does not keep track by activity internally, so we need to find all rules and sum all files locked by each rule"

Reply:
I believe we should put this comment on line 145 ```totalusage = sum(getRuleQuota(rucioClient, rule['id']) for rule in rules if rule['state'] in valid_states)``` not lines 126-129

Reply:
looks like i clicked on wrong line, sorry. Mostly I'd like a comment which explains why we handle differently activity=None and activity=something